tests/playwright/vite.config.js

```diff
@@ -36,7 +37,7 @@ const config = {
       '/@/': join(PACKAGE_ROOT, 'src') + '/',
     },
   },
-  plugins: [typescript()],
+  plugins: [typescript(), dtsPathAlias()],
```

Comment:
remark: in https://github.com/buddyron/rollup-plugin-dts-path-alias?tab=readme-ov-file#usage, the documentation says to

>  Apply the path alias plugin before generating .d.ts files

<img width="764" height="247" alt="Image" src="https://github.com/user-attachments/assets/856011a4-c1ef-4183-8025-5eb5d71d29a3" />

And here we do it after, I don't know if it is an issue.